### DISCUSSION TITLE
Secure login on web interface

### DISCUSSION THREAD

**Discussion by drag095** [Geektoolkit/Dynaframe3#153] [Ideas]
Hello,
I am using Dynaframe3, version 2.23 and i think it's working great and the command panel it's well structured, but the web interface doesn't have a secure login, which means that everyone who knows the IP address can connect and modify settings. It is possible to add a login screen so only the intended user can access the webpage? Also, it is possible to synchronize two or more pi's to play simultaneously the same video file?

**Comment by Geektoolkit** [comment]
it is possible to use the sync frame feature to sync two or more frames, however the timing is slightly off in this version. I don't plan myself on adding a login page to this, it was meant to be a 'at home on a local network' device, so the auth is based on the wifi /creds of the house its in.
  I am working on a new version I'm calling 'Dynaframe pro' which will be patreon supported to work on more professional features as there are a number of companies that are looking to use this, and asking for features that are going to take a bit more time/work. I'll be announcing that shortly, I'm finishing up fit and fniish on that build now. it also includes a reworked sync engine, database backend, themeing, and other 'pro' features. It'll also be better supported and have continuous feature development as well as the ability to easily deploy multiple frames, download content from a webstore and more. But I don't plan on backporting all of that work to this..I'll leave this for the community to build on.
